Abracadabra/abracadabra_web_app/src/components/QuestionBody.js

```diff
@@ -46,9 +46,6 @@ function Title({ question, subject, search, searchLength }) {
                   <div className="col-sm-2">
                     {NrOfAnswers(q.numberOfAnswers)}
                   </div>
-                  <div className="col-sm-1">
-                  <p>{q.upvotes - q.downvotes}</p>
```

Comment:
Was this always displaying 0? "Removed useless upvote counter on QuestionBody. it was always 0"

Reply:
Yea it was